app/Models/JobPoster.php

```diff
@@ -154,6 +155,11 @@ public function job_poster_translations() {
         return $this->hasMany(\App\Models\JobPosterTranslation::class);
     }
 
+    public function screening_plans()
```

Comment:
Are we sticking to camelCase or snake_case for naming things in PHP?

Reply:
Eloquent model attributes and relations are snake_case, to match the database column names. Other php variables and functions should be camelCase, I think.